docs/Bigtable-batchsink.md

```diff
@@ -40,3 +40,26 @@ Full list of allowed properties: https://cloud.google.com/bigtable/docs/hbase-cl
 **Service Account File Path**: Path on the local file system of the service account key used for
 authorization. Can be set to 'auto-detect' when running on a Dataproc cluster.
 When running on other clusters, the file must be present on every node in the cluster.
+
+Data Type Mappings from CDAP to BigQuery
+----------
+The following table lists out different CDAP data types, as well as the 
+corresponding BigQuery data type for each CDAP type, for updates and upserts.
+
+| CDAP type | BigQuery type |
+|-----------|---------------|
+| long      | int64         |
```

Comment:
should include all CDAP types, seems to be missing int, float, and union. I believe int -> int64, float -> float64, and union is not supported.

Also, were you able to verify that this information is correct? The stuff I put in the jira was off the top of my head.

Reply:
How can I verify these types? Is there a master list somewhere? Or should I create a BQ source and see how the types are loaded into CDAP?

Reply:
this is the sink. You should verify that this information is correct by running pipelines